package.json

```
@@ -1,27 +1,30 @@
 {
   "name": "visenze-tracking-javascript",
-  "version": "1.2.1",
+  "version": "2.0.0",
```

Let's make it rc version first, so we are certain all the integrations work.
```suggestion
  "version": "2.0.0-rc.0",
```